fastlane/Fastfile

```diff
@@ -39,19 +39,26 @@ platform :android do
         branch = git_branch
         # If the branch is master, this means new updates were merged into master
         # Then publish the artifacts to bintray
-        if branch == 'origin/master'
-          publish_artifacts
+        if branch == 'master'
+          publish_artifacts(
+            snapshot: false
+          )
+
+          # TODO: publish Snapshots with snapshot = true
         end
       end
     end
   end
 
   desc "publish artifacts to bintray"
-  lane :publish_artifacts do
+  lane :publish_artifacts do |options|
     if ENV['SONATYPE_TOKEN_USER'] && ENV['SONATYPE_TOKEN_PASSWORD']
       # Upload to bintray
       gradle(
-        task: "vimeo-networking:bintrayUpload"
+        task: "vimeo-networking:bintrayUpload",
+        properties: {
+          "snapshot" => options[:snapshot]
+        }
```

Comment:
Just found this guy: https://github.com/bintray/gradle-bintray-plugin#maven-central-sync think we can use this instead?